api/v1beta1/cloudstackmachine_types.go

```diff
@@ -52,14 +52,14 @@ type CloudStackMachineSpec struct {
 
 	// Optional affinitygroupids for deployVirtualMachine
 	// +optional
-	AffinityGroupIds []string `json:"affinitygroupids,omitempty"`
+	AffinityGroupIDs []string `json:"affinitygroupids,omitempty"`
```

Comment:
Linter prefers ID almost everywhere apparently.

Reply:
It's going to take me a while to get used to that.  I've spent many years following Google's Java style guide, which says to use Url and Id.  But the Golang wiki says to use URL and ID.  https://github.com/golang/go/wiki/CodeReviewComments#initialisms